ci/verify_examples.sh

```diff
@@ -11,6 +11,8 @@ WARNINGS=()
 # Sandboxes listed here should be regarded as broken(/untested) until whatever
 # is causing them to flake is resolved!!!
 FLAKY_SANDBOXES=(
+    # no device available in CI
+    dlb
```

Comment:
as this doesnt run at all (rather than fail flakily) it should be excluded here https://github.com/envoyproxy/envoy/blob/fd1058b9bd6a512118ef88e6c09eea9a3555696f/ci/do_ci.sh#L963